test_util/launch_configs/onprem-from-info.yaml

```diff
@@ -0,0 +1,15 @@
+---
+# no deployment_name provided because it was set in the cloud_cluster
+deployment_type: onprem
+installer_url: https://downloads.dcos.io/dcos/testing/master/dcos_generate_config.sh
+provider: cloud_cluster
+provider_info_json_path: /foo/bar/important/path
```

Comment:
This is currently referenced as `provider_info_path` in the docs. Update either here or there to be consistent.